src/main/java/com/ilich/repository/AdvertInfoRepository.java

```diff
@@ -0,0 +1,65 @@
+package com.ilich.repository;
+
+import com.ilich.mapper.AdvertInfoRowMapper;
+import com.ilich.model.AdvertInfoData;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.PreparedStatementCreator;
+import org.springframework.jdbc.support.GeneratedKeyHolder;
+import org.springframework.jdbc.support.KeyHolder;
+import org.springframework.stereotype.Repository;
+
+import javax.sql.DataSource;
+import java.sql.PreparedStatement;
+import java.util.Objects;
+
+import static java.sql.Statement.RETURN_GENERATED_KEYS;
+
+
+@Repository
+public class AdvertInfoRepository {
+
+    private static final String SELECT = "SELECT * FROM adverts_info WHERE id_adverts_info = ?";
```

Comment:
Be more specific in constant naming - what does the select do - include it into the name. Also, can you use `named queries` instead? Also, look at Spring Data, it makes sense to try it out for you